__tests__/app.test.js

```diff
@@ -66,6 +66,84 @@ describe("GET: /api/articles/:article_id", () => {
                 expect(response.body.msg).toBe("Article does not exist");
             });
     });
+    test('PATCH: 200 update votes on an article', () => {
+        const votes = {inc_votes: 10}
```

Comment:
You may also wish to test this with a negative number and see that it reduces the votes 😊


Overall, really good Nicola, just a couple minor tweaks